asgard/handler.cpp

```diff
@@ -240,9 +242,26 @@ Handler::Handler(const Context& context) : graph(context.graph),
                                            valhalla_service_url(context.valhalla_service_url) {
 }
 
+void Handler::log_elapsed_time(const std::chrono::high_resolution_clock::time_point& start, const std::string& request_id) const {
+    auto elapsed_time = duration_cast<milliseconds>(high_resolution_clock::now() - start);
+
+    LOG_INFO("It took " + std::to_string(elapsed_time.count()) + " ms to handle request: " + request_id);
+}
+
 pbnavitia::Response Handler::handle(const pbnavitia::Request& request) {
+
+    auto start = high_resolution_clock::now();
+    std::string request_id;
```

Comment:
```suggestion
    std::string request_id = "unknown_request_id;
```
I always fear to have an uninitialized variable when I see a variable declared without assignment